chat_exporter/__init__.py

```diff
@@ -1,6 +1,6 @@
 from chat_exporter.chat_exporter import export, raw_export, quick_export, link, quick_link
 
-__version__ = "2.6.1"
+__version__ = "2.7.0"
```

Comment:
should be 2.6.2

Reply:
`2.7.0` will be the next version - however as mentioned on Discord; I wouldn't release this standalone. 

Also, I've found it's often best to leave versioning for the maintainers for open source projects - many people have opinions on versioning, so it's best to just leave it normally. This is fine as is though.